superset/views/core.py

```diff
@@ -748,9 +748,10 @@ def shortner(self):
         obj = models.Url(url=url)
         db.session.add(obj)
         db.session.commit()
+        scheme = request.scheme
```

Comment:
Why do we need this temporary variable? I sense it adds for more obfuscation and thus would include this on line 754.